app/src/main/kotlin/com/simplemobiletools/keyboard/activities/SettingsActivity.kt

```diff
@@ -112,7 +112,7 @@ class SettingsActivity : SimpleActivity() {
             val items = arrayListOf(
                 RadioItem(LANGUAGE_ENGLISH_QWERTY, getKeyboardLanguageText(LANGUAGE_ENGLISH_QWERTY)),
                 RadioItem(LANGUAGE_ENGLISH_QWERTZ, getKeyboardLanguageText(LANGUAGE_ENGLISH_QWERTZ)),
-		RadioItem(LANGUAGE_ENGLISH_DVORAK, getKeyboardLanguageText(LANGUAGE_ENGLISH_DVORAK)),
+                RadioItem(LANGUAGE_ENGLISH_DVORAK, getKeyboardLanguageText(LANGUAGE_ENGLISH_DVORAK)),
```

Comment:
Fixed "replace tabs with spaces please"

_Originally posted by @tibbi in https://github.com/SimpleMobileTools/Simple-Keyboard/pull/46#discussion_r878930653_